confer.go

```diff
@@ -178,6 +184,19 @@ func (manager *Config) Get(key string) interface{} {
 	return v
 }
 
+// NewConfigSubset returns a new config based on
+func (manager *Config) NewConfigSubset(prefix string) *Config {
+	configSubset := &Config{}
+	*configSubset = *manager
```

Comment:
No surprise to me :-).This was intentional, as I only want to modify the subset of config passed. That's why I chose to prefix the function name with "new".  I can change it to a reference, but the intent was to make a copy of the subset.